manager/main.go

```diff
@@ -55,7 +56,7 @@ func init() {
 
 func run(namespace string, metricsAddr string, enableLeaderElection bool,
 	enableApplicationController, enableBlueprintController, enablePlotterController bool) int {
-	setupLog.Trace().Msg("creating manager")
+	setupLog.Trace().Msg("creating manager. based on git commit: " + gitCommit)
```

Comment:
I don't see what value is assigned to this variable

Reply:
@shlomitk1 , it is done in the Make file. 
here is the output of the `make run` rule:
```
make run                                                                        
go run -ldflags "-X main.gitCommit=ec73bb93fb991b2885e0cf7ff31744c9981a2ee7" ./main.go --enable-all-controllers --metrics-bind-addr=0
Trouble reading verbosity. Found -1 . Using trace as default
10:10PM DBG ../pkg/logging/logging.go:111 > Logging verbosity level is trace Setup=main
10:10PM TRC main.go:59 > creating manager. based on git commit: ec73bb93fb991b2885e0cf7ff31744c9981a2ee7 Setup=main
10:10PM INF main.go:66 > Application namespace:  Setup=main
10:10PM INF main.go:83 > Manager client rate limits: qps = 5 burst=10 Setup=main
```